tests/unittests/Test-r2hdm.cpp

```diff
@@ -82,6 +93,19 @@ TEST_CASE("Checking EWPT for R2HDM", "[r2hdm]")
   }
 }
 
+TEST_CASE("Checking sign of Sin and Cos of beta", "[r2hdm]")
+{
+  using namespace BSMPT;
+  using namespace Models;
+  Class_Potential_R2HDM point1, point2;
+  point1.set_gen(example_point_R2HDM);
+  point2.set_gen(example_point_R2HDM_negTanBeta);
+
+  REQUIRE(((point1.C_CosBeta >= 0) and (point2.C_CosBeta >= 0) and
```

Comment:
Please split this up in four seperate requirements. This way we have an better overview what fails

Reply:
Done in 9723d3b.